src/sentry_plugins/sessionstack/plugin.py

```diff
@@ -136,18 +136,22 @@ def get_config(self, project, **kwargs):
         return configurations
 
     def get_event_preprocessors(self, data, **kwargs):
-        def preprocess_event(event):
-            extra = event.get('extra') or {}
-            sessionstack_context = extra.get('sessionstack') or {}
+        if data.get('platform') != 'javascript':
+            return []
+
+        project = Project.objects.get_from_cache(id=data.get('project'))
```

Comment:
Can you move this one down to be the last check? This will still cause a network roundtrip and the rest does not.